content/blog/2020-08-19-train-a-sequence-model-with-poutyne/article_notebook.ipynb

```diff
@@ -0,0 +1,940 @@
+{
```

Comment:
Friendly reminder to make sure the Jupyter Notebook and article match eachother before final submission.

Also, as stated below, the Jupyter cannot be ran in Colab due to RAM requirements.